canonical-web-and-design/global-nav
# v3

## Done

- Updated global-nav to work specifically with the Vanilla navigation pattern, rather than as a standalone element

## QA

- View the demo
- On desktop, see that the "All Canonical" nav item is present, and that the dropdown appears when clicked
- On mobile, see that "All Canonical" is _not_ present, and instead "Products", "Also From Canonical", "Resources" and "About" dropdowns are visible, and work
- Accessibility: the dropdown should comply with the Vanilla a11y docs: https://vanillaframework.io/docs/patterns/navigation/accessibility

Fixes: https://github.com/canonical-web-and-design/web-squad/issues/4989
